src/execution/operator/projection/physical_tableinout_function.cpp

```diff
@@ -52,14 +54,30 @@ unique_ptr<GlobalOperatorState> PhysicalTableInOutFunction::GetGlobalOperatorSta
 	return std::move(result);
 }
 
+void PhysicalTableInOutFunction::PrepareOrdinality(DataChunk &chunk, idx_t &ord_index, bool &ord_reset) const {
+	const idx_t ordinality = chunk.size();
```

Comment:
This looks like it wouldn't work with datasets over 2048 tuples?
If we set ordinality to chunk.size()

Reply:
I might have given this variable a confusing name, it is only used to check whether or not the chunk is empty and then to fill the chunk with inidices using the (duckdb)Vector::Sequence()-function, see comment below

Reply:
I think I follow, but since you set `ordinality` to chunk.size(), which will never exceed past 2048 (STANDARD_VECTOR_SIZE) this looks like a bug

Is this just an intermediate or does the result of this SequenceVector directly show up in the final result set?